### PR TITLE
Updating some flows and tackling some small bugs

### PR DESCRIPTION
**Pull request by joshuaGlass808**

### CODE REVIEW

File: resources/js/Pages/Login.vue [hunk 1]
```diff
@@ -75,12 +75,19 @@
...
-            _token: this.csrfToken
+            _token: this.csrfToken,
+            onSuccess: (res: any) => {
+              if ('error' in res.props) {
+                alert(res.props.error);
+              }
+            }
           });
+
+          console.log(test, 'test');
```
Comment (jlrusso) [inline resources/js/Pages/Login.vue]: rm

File: resources/js/Pages/Login.vue [hunk 2]
```diff
@@ -75,12 +75,19 @@
...
           const payload: string = this.getValidatedPayload();
-          this.$inertia.post('/login', payload, {
+          const test = this.$inertia.post('/login', payload, {
             replace: false,
             preserveState: true,
             preserveScroll: false,
-            _token: this.csrfToken
+            _token: this.csrfToken,
+            onSuccess: (res: any) => {
+              if ('error' in res.props) {
```
Comment (jlrusso) [inline resources/js/Pages/Login.vue:84]: i'm going to start using this instead of Object.hasOwnProperty()

File: resources/js/Pages/Login.vue [hunk 3]
```diff
@@ -75,12 +75,19 @@
       loginHandler(): void {
         try {
           const payload: string = this.getValidatedPayload();
-          this.$inertia.post('/login', payload, {
+          const test = this.$inertia.post('/login', payload, {
```
Comment (jlrusso) [inline resources/js/Pages/Login.vue]: rm

**Review by jlrusso — APPROVED** [review]

File: resources/js/Pages/Login.vue (hunk 2 above)
Comment (joshuaGlass808) [reply resources/js/Pages/Login.vue:84]: I wasn't sure if I should use that or hasOwnProperty, but using the in operator just felt cooler lol.

**Pull request merged**